app/Model/Infrastructure/Repositories/CandidateRepository.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Infrastructure\Repositories;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Model\Candidate\Candidate;
+use Model\Candidate\CandidateFunction;
+use Model\Candidate\FunctionNotFound;
+use Model\Candidate\Repositories\ICandidateRepository;
+use Model\DTO\Cashbook\SkautisCandidate;
+use function assert;
+
+final class CandidateRepository extends AggregateRepository implements ICandidateRepository
+{
+    /**
+     * @param SkautisCandidate[] $candidates
+     *
+     * @throws ORMException
+     * @throws OptimisticLockException
+     */
+    public function saveCandidates(array $candidates) : void
```

Comment:
To ukládání má dost vnitřní logiky, mohl bys pls doplnit testy?

Reply:
done